src/dlload.c

```diff
@@ -403,7 +404,7 @@ JL_DLLEXPORT void *jl_load_dynamic_library(const char *modname, unsigned flags,
     return handle;
 }
 
-JL_DLLEXPORT int jl_dlsym(void *handle, const char *symbol, void ** value, int throw_err) JL_NOTSAFEPOINT
+JL_DLLEXPORT int jl_dlsym(void *handle, const char *symbol, void ** value, int throw_err, int no_deps) JL_NOTSAFEPOINT
```

Comment:
It could be nice from a readability perspective to not have the argument represent a negation, as the double negative in `!no_deps` is (subjectively) less intuitive. That would also make it consistent with the `throw_err` argument, which isn't e.g. `dont_throw_err`. Anyway, just my drive-by $0.02, now back into the shadows I go...